tests/dragonfly/replication_test.py

```diff
@@ -1838,31 +1838,62 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     replica.stop()
 
 
+# note: please be careful if you want to change any of the parameters used in this test.
+# changing parameters without extensive testing may easily lead to weak testing case assertion
+# which means eviction may not get triggered.
 @pytest.mark.asyncio
 @pytest.mark.slow
 async def test_eviction_propagation(df_local_factory, df_seeder_factory):
-    master = df_local_factory.create(proactor_threads=1, cache_mode="true", maxmemory="290mb")
+    master = df_local_factory.create(
+        proactor_threads=1, cache_mode="true", maxmemory="267mb", enable_heartbeat_eviction="false"
+    )
     replica = df_local_factory.create(proactor_threads=1)
     df_local_factory.start_all([master, replica])
 
     c_master = master.client()
     c_replica = replica.client()
 
-    seeder = df_seeder_factory.create(port=master.port, val_size=1000, stop_on_failure=False)
-
     await c_master.execute_command("DEBUG POPULATE 6200000")
```

Comment:
I still dont think you need to use this big number of keys.
When you start dragonfly with one shard you will have 6720 slots
you can run debug populate for 6500 keys which will almost fill the number of slots in dragonfly and to make sure you will get to your limit you can change the value of the items you populate 256MB/650 items -> set 40000 bytes for value in debug populate and you will get very close to the memory limit. than inserting new data with seeder will trigger the eviction

Reply:
if you want to use the seeder to do so, the memory consumption is really not calculated this way. 

For example: 

A seeder created using 

```
seeder = df_seeder_factory.create(port=master.port, keys=256, val_size=1000, stop_on_failure=False)
```

Will not result in 256MB DF memory usage, but just a few MB..  maybe I missed something, have you tried?

Reply:
I suggested to fill the database with debug populate not with the seeder to be close to the 256MB
If you run `debug populate 6000 key 44000` you will add 6000 keys with value size of 44000 which will result in used memory : used_memory_human:234.67MiB
This way you can fill the data base with low number of keys reaching almost maxmemory size

Reply:
wait maybe I did not understand you question you are asking how to do this with the seeder.
you should do seeder.run(target_deviation=0.1) to fill the db

Reply:
now it only uses seeder, please take another look.